docs/development/security/consent.md

```diff
@@ -0,0 +1,69 @@
+# Samtykkeerklæring
+
+Anmodning om samtykke til indsamling af almindelig personlig information. Hvis data subjektet er under 18 år, er det vigtigt at samtykkeerklæring indhentes fra den pågældendes værge.
+
+Oplysninger om vores behandling af dine personoplysninger mv.
+
+## 1. Vi er den dataansvarlige – hvordan kontakter du os?
+
+Girafs Venner er dataansvarlig for behandlingen af de personoplysninger, som vi har modtaget om dig. Du finder vores kontaktoplysninger nedenfor.
+
+* Navn: Girafs Venner
+* Adresse: Selma Lagerløfs vej 300, 9220 Aalborg Øst
+* CVR-nr.: 40519025
+* Telefon: 40 89 21 56
+* Mail: ulrik@cs.aau.dk
+
+## 2. Formålene med og retsgrundlaget for behandlingen af dine personoplysninger
+
+Vi behandler dine personoplysninger til følgende formål:
+
+* Formålet med behandlingen af personlige oplysninger er udelukkende at tilvejebringe et kommunikationsværktøj til autistiske børn, og medarbejdere til den udbudte institution. Behandlingen er således kun med formål at skabe en personaliseret interaktion, mellem systemet og det enkelte barn, ved at have en systembruger.
+
+* Legitime interesser, der forfølges med behandlingen.
+
+Som nævnt ovenfor sker vores behandling af dine personoplysninger på baggrund af interesseafvejningsreglen i databeskyttelsesforordningens artikel 6, stk. 1, litra f. De legitime interesser, der begrunder behandlingen, er at hjælpe autistiske børn med at kommunikere og skabe struktur i hverdagen gennem et kommunikationsværktøj.
+
+## 3. Kategorier af personoplysninger
+
+Vi behandler følgende kategorier af personoplysninger om dig: 
+Identifikationsoplysninger:
+
+* Almindelige personoplysninger
+
+## 4. Hvor dine personoplysninger stammer fra
+
+Personoplysningerne stammer fra registreringen af brugeren i applikationen.
+
+## 5. Opbevaring af dine personoplysninger
+
+Personlige oplysninger slettes 1 år efter, at brugeren er erklæret inaktiv, hvilket afhængigt af institutionen enten kan være et aktivt valg foretaget af institutionen eller som resultat af manglende anvendelse af systemet.
```

Comment:
Does it, though?

Reply:
Hvis ikke - Så skal vi nok lave et issue på at få implementeret dette.

Reply:
Ja, det er der også en issue på. Og i tilfælde af at der findes en anden løsning, så skal det her selvfølgeligt ændres. Det er ligeså meget et bud på hvordan man kunne gøre, fordi at vi har diskuteret det, også med Ulrik, men implementationen mangler stadigt. De her dokumenter skal generelt revideres så snart vi tilføjer behandling af personlig information eller ændre organisationsstruktur, derfor er det svært at lave en helt korrekt version før at det rent faktisk skal udbydes.